packages/bot-web-ui/src/components/flyout-block-group.jsx

```diff
@@ -26,39 +29,42 @@ const FlyoutBlockGroup = ({ onInfoClick, block_node, is_active, should_hide_disp
     );
 
     return (
-        <div className={classNames('flyout__item', { 'flyout__item--active': is_active })}>
-            {!should_hide_display_name && (
-                <>
-                    <div className='flyout__item-header'>
-                        <div
-                            className={classNames({
-                                'flyout__item-label--small': is_variables_get,
-                                'flyout__item-label--bold': !is_variables_get,
-                            })}
-                        >
-                            {display_name}
-                        </div>
-                        {!is_variables_get && AddButton()}
-                    </div>
-                    <div className='flyout__item-description'>
-                        {description}
-                        {onInfoClick && (
-                            <a
-                                id={display_name.replace(/\s/gi, '-')}
-                                className='flyout__item-info'
-                                onClick={onInfoClick}
+        <>
+            {is_variables_set && <Hr />}
+            <div className={classNames('flyout__item', { 'flyout__item--active': is_active })}>
+                {!should_hide_display_name && (
+                    <>
+                        <div className='flyout__item-header'>
+                            <div
+                                className={classNames({
+                                    'flyout__item-label--small': is_variables_get,
+                                    'flyout__item-label--bold': !is_variables_get,
+                                })}
                             >
-                                {localize('Learn more')}
-                            </a>
-                        )}
-                    </div>
-                </>
-            )}
-            <div className='flyout__block-workspace__header'>
-                <FlyoutBlock should_center_block={true} block_node={block_node} should_hide_display_name />
-                {is_variables_get && AddButton()}
+                                {display_name}
+                            </div>
+                            {!is_variables_get && AddButton()}
+                        </div>
+                        <div className='flyout__item-description'>
+                            {description}
+                            {onInfoClick && (
+                                <a
+                                    id={display_name.replace(/\s/gi, '-')}
+                                    className='flyout__item-info'
+                                    onClick={onInfoClick}
+                                >
+                                    {localize('Learn more')}
```

Comment:
it's better if you use the Localize component when you are working with jsx if you want to pass localize as a component attribute you can use the localize function

Reply:
Actually, I didn't make any change in this part but refactor that.